test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -584,6 +584,7 @@ func testWorkspacesVirtualWorkspaces(t *testing.T, standalone bool) {
 		// write kubeconfig to disk, next to kcp kubeconfig
 		kcpAdminConfig, _ := server.RawConfig()
 		var baseCluster = *kcpAdminConfig.Clusters["base"] // shallow copy
+		baseCluster.Server = fmt.Sprintf("%s/clusters/system:admin", baseCluster.Server)
```

Comment:
alternatively, we could add something to a handler but having a cluster name here is much cleaner.